sympy/physics/continuum_mechanics/arch.py

```diff
@@ -146,56 +151,61 @@ def apply_load(self,order,label,x1,mag,x2=None,angle=None):
                 The angle in degrees, the load vector makes with the horizontal
                 in the counter-clockwise direction.
 
+        Examples
+        ========
+        For applying distributed load
+
+        >>> from sympy.physics.continuum_mechanics.arch import Arch
+        >>> a = Arch((0,0),(10,0),crown_x=5,crown_y=5)
+        >>> a.apply_load(0,'A',x1=3,x2=5,mag=10)
+
+        For applying point/concentrated_loads
+
+        >>> from sympy.physics.continuum_mechanics.arch import Arch
+        >>> a = Arch((0,0),(10,0),crown_x=5,crown_y=5)
+        >>> a.apply_load(-1,'B',x1=2,mag=15,angle=45)
+
         """
         if label in self._loads:
             raise ValueError("load with the given label already exists")
 
-        self._loads[label] = mag
         if order == 0:
             if not x2 or x2<x1:
                 raise KeyError("provide x2 greater than x1")
 
             if x1>self._right_support[0] or x2<self._left_support[0]:
                 raise ValueError(f"loads must be applied between {self._left_support[0]} and {self._right_support[0]}")
-            self._distributed_loads[label] = (x1,x2)
-        if order == 1:
+            self._distributed_loads[label] = {'start':x1, 'end':x2, 'f_y': -mag}
```

Comment:
I am having some trouble understanding how you have implemented directions for loads. What does the negative sign here imply? 
For a distributive and concentrated load both facing down is this the right way to add the loads?
```
>>> a.apply_load(0,'A',x1=20,x2=40,mag=-4)
>>> a.apply_load(-1,'B',x1=8,mag=150,angle=90)
```
I ran the method on the arch from example 6.2 from [here](https://temple.manifoldapp.org/read/a72c8590-5028-4cb4-b3a8-0fb6f9e4026a/section/0ad83309-27c7-4341-aa13-b057374a134c#sec6-1) and it's swapping the signs of `R_A_x` & `R_B_x` and hence not matching with the correct answers.

Reply:
currently all the forces component in x and y direction are in positive x and postive y direction , the imaginary x axis I'm taking is +ve on the right hand side and y axis is vertically up . So it might be the reason why R_A_x and R_B_x might be having opposite directions. 

It is also the reason why I'm using a negative sign with the magnitude in the distributed load. During the initial coding period of the solve method , I was testing the results for downward forces which is usually the case so I put up a negative sign to match the sign convention.